include/sfx.h

```diff
@@ -323,7 +323,7 @@ enum Sfx {
     SFX_ARROW_SHOT_C,
     SFX_ARROW_SHOT_D,
     SFX_SKELETON_DEATH_A,
-    SFX_SKELETON_DEATH_B,
+    SFX_SKELETON_DEATH_B, // Slinger
     SFX_SKELETON_DEATH_C,
```

Comment:
C looks to be used for bone scimitar and skeleton. A doesn't appear to be used in our code just yet. I just made a comment here in case we want to rename this down the track